### ISSUE TITLE
Add screenshots to housekeeping

### ISSUE THREAD

**Issue by danirabbit** [elementary/settings-daemon#55]
### Problem

I always end up taking a ton of screenshots and I just have like hundreds of files that I definitely don't need anymore

### Proposal

Would be great to add the screenshots directory to housekeeping

**Comment by Antolius** [comment]
Would this behavior require a new toggle in the _Housekeeping_ view of the_System Settings_, or would it be enabled by one of existing options (_old temporary files_ perhaps)?

**Comment by danirabbit** [comment]
@Antolius Yeah I think it should be a new checkbox since it's a different location

**Comment by Antolius** [comment]
Cool! I'd like to give implementing this a try.

Btw, @danirabbit would you mind me looking into #47 as well? (If you are OK with it in principal I can open a separate PR for that.)
